packages/manager/apps/pci-private-registry/src/components/Plan.component.tsx

```diff
@@ -20,28 +24,23 @@ export default function PlanComponent({
 
   const { data: catalog, isPending } = useCatalog();
 
+  const { getFormattedCatalogPrice: monthly } = useCatalogPrice(2);
```

Comment:
Why rename the function ? The new name appears as a variable instead of a function